Introduce Ansible plugin in Jetbrains section of doc (#72465)
ansible/ansible: docs/docsite/rst/community/other_tools_and_programs.rst

An open-source Community edition and closed-source Enterprise edition, integrated development environments based on IntelliJ's framework including IDEA, AppCode, CLion, GoLand, PhpStorm, PyCharm and others. Useful JetBrains platform plugins include:

> * `Ansible <https://plugins.jetbrains.com/plugin/14893-ansible>`_ - Ansible plugin for Jetbrains IDEs. It provides auto-completion, role name suggestion and other  handy features to work with playbooks and roles. 

Reviewer: ```suggestion
* `Ansible <https://plugins.jetbrains.com/plugin/14893-ansible>`_ - general Ansible plugin provides auto-completion, role name suggestion and other handy features for working with playbooks and roles.

```
